### PR TITLE
Bump server version to 6.0

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Adds HeavyDB 6.0 and drops CI support for versions 5.7 and 5.8.

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
@pearu, do you happen to know why CI is failing here?

**Comment by guilhermeleobas** [comment]
Seems to be all good now

File: .github/workflows/rbc_test.yml [hunk 1]
```diff
@@ -101,21 +101,21 @@ jobs:
...
+        #     numba-version: '0.55'
+        #     heavydb-version: dev
+        #     docker-image: heavyai/core-os-cpu-dev:latest
+        #     heavydb-from: docker
+        #   - os: ubuntu-latest
+        #     python-version: '3.9'
+        #     numba-version: '0.54'
+        #     heavydb-version: dev
+        #     docker-image: heavyai/core-os-cpu-dev:latest
+        #     heavydb-from: docker
```
Comment (pearu) [inline .github/workflows/rbc_test.yml:118]: What is the plan with this commented-out section? Is `heavyai/core-os-cpu-dev:latest` available?

**Comment by pearu** [comment]
Why the CI tests are not running here?

**Comment by guilhermeleobas** [comment]
Not sure. I've manually triggered the tests.

File: .github/workflows/rbc_test.yml (hunk 1 above)
Comment (guilhermeleobas) [reply .github/workflows/rbc_test.yml]: It is available now

**Comment by guilhermeleobas** [comment]
update: Tests are running now

File: .github/workflows/rbc_test.yml (hunk 1 above)
Comment (guilhermeleobas) [reply .github/workflows/rbc_test.yml]: The `-dev` images are not available.

**Review by pearu — APPROVED** [review]
LGTM!

**Pull request merged**